--- solr.in.sh	2020-11-16 14:44:44.000000000 -0800
+++ solr.in.sh.mod	2020-11-19 13:19:43.000000000 -0800
@@ -28,7 +28,7 @@
 #SOLR_STOP_WAIT="180"
 
 # Increase Java Heap as needed to support your indexing / query needs
-#SOLR_HEAP="512m"
+SOLR_HEAP="1g"
 
 # Expert: If you want finer control over memory options, specify them directly
 # Comment out SOLR_HEAP if you are using this though, that takes precedence
