--- config.xml	Mon Jun 10 11:29:30 2019
+++ config - Copy.xml	Tue Jun 18 15:55:19 2019
@@ -117,13 +117,13 @@
          NB you may use ipv4 or ipv6 addresses as appropriate
          So called hybrid addressing is also supported and when an ipv4 address is checked on a ipv6 capable server the address checked will be the embedded ipv4 in ipv6 format (ipv6 addresses are ignored if server is only ipv4 capable)
          e.g. 192.168.5.109 is equivalent to ::ffff:192.168.5.109 (hybrid form of embedded ipv4 address) is equivalent to ::ffff:c0a8:56d so any of these forms may be used below (typically the ipv4 form is just simpler to type and when DCHCP rather than SLAAC used will be sufficient)
-     -->
     <property name="only_accept_ip">
         <list>
           <value>127.0.0.1</value>
           <value>::1</value>
          </list>
     </property>
+     -->
     
     <!-- MaxRequestTimeInSeconds - by default we restrict this admin worker to operations which take less than an hour
          At some point we will change this to a value such as 1s since the idea is to avoid long running operations on admin - however for backward compatability with existing tools we currently support
